seatunnel-connectors-v2/connector-file/connector-file-base/pom.xml

```diff
@@ -158,6 +159,13 @@
             <artifactId>jaxen</artifactId>
             <version>${jaxen.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>easyexcel</artifactId>
+            <version>${easyexcel.version}</version>
+        </dependency>
```

Comment:
As we all know, easyexcel is no longer maintained. It doesn't seem good to introduce it at this time. We can try other alternatives, such as [fastexcel](https://github.com/dhatim/fastexcel). There are also [reports](https://stackoverflow.com/questions/69914445/apache-poi-large-excel-export-is-slow/69961916#69961916) online that it is faster than easyexcel. What do you think? cc @hailin0

Reply:
or easyexcel-plus？

Reply:
I will give it a try

Reply:
> or easyexcel-plus？

 easyexcel-plus was only on GitHub last night, and I haven't seen it in the maven repository yet

Reply:
> As we all know, easyexcel is no longer maintained. It doesn't seem good to introduce it at this time. We can try other alternatives, such as [fastexcel](https://github.com/dhatim/fastexcel). There are also [reports](https://stackoverflow.com/questions/69914445/apache-poi-large-excel-export-is-slow/69961916#69961916) online that it is faster than easyexcel. What do you think? cc @hailin0

I tried using fastexcel, but there is a problem with its xls support for excel97-2003
![image](https://github.com/user-attachments/assets/66f02391-df27-47bf-a786-f1faffe1ce8e)

Reply:
oh, let's add an option to configure the excel parse engine, default POI, support POI and easyexcel at now. So we can implement other engine in the future.

Reply:
Okay

Reply:
> oh, let's add an option to configure the excel parse engine, default POI, support POI and easyexcel at now. So we can implement other engine in the future.

Will there be any conflict between poi versions?

Reply:
we can use poi version in easyexcel for now.

Reply:
> we can use poi version in easyexcel for now.

What's the meaning of it ？

Reply:
we can set poi version in https://github.com/apache/seatunnel/pull/8064/files#diff-1fb7018c76d4a71faced6c343bcbff3a7d20464c8c9f80338fa2a50e57fae254R142 consistent with that in easyexcel.

Reply:
I added a configuration item for excel reading to select the engine for excel reading. POI is used by default. EaseExcel can be used through configuration. The configuration item name is excel_engine.

Reply:
![image](https://github.com/user-attachments/assets/276e7836-bebd-46b3-8e14-704a9ec20d5e)
In the CI phase, there are errors in the test cases of other modules, causing CI to fail

Reply:
> ![image](https://private-user-images.githubusercontent.com/14970783/390229768-276e7836-bebd-46b3-8e14-704a9ec20d5e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2NzkzMzYsIm5iZiI6MTczMjY3OTAzNiwicGF0aCI6Ii8xNDk3MDc4My8zOTAyMjk3NjgtMjc2ZTc4MzYtYmViZC00NmIzLThlMTQtNzA0YTllYzIwZDVlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI3VDAzNDM1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjOWVkY2JhODVhNjI0YWY1NzRkMzFhYmU4MzUxNWUwYjljM2RiZTQ1MmM5NmNhMGVkNmNkNWVjZTUzYzFmZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rIjNHXp-7cfn17EzPEX4SoxxDsZ24PjxFTQF-ZIm6mU) In the CI phase, there are errors in the test cases of other modules, causing CI to fail

retry, many reasons can be resolved by retrying[/dog]

Reply:
Finally succeeded